zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/launcher/SparkInterpreterLauncher.java

```diff
@@ -272,22 +272,34 @@ private String detectSparkScalaVersion(String sparkHome, Map<String, String> env
     builder.environment().putAll(env);
     File processOutputFile = File.createTempFile("zeppelin-spark", ".out");
```

Comment:
You can use the following function here.
`Files.createTempFile(...)`

https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#createTempFile-java.lang.String-java.lang.String-java.nio.file.attribute.FileAttribute...-